source/support/FAQ/services-faq.rst

```diff
@@ -62,7 +72,7 @@ While we are intent on providing the most friendly experience possible to our cu
 
 I want to understand in depth how a Service works and it's available configuration options.  Where can I go to learn more?
```

Comment:
Here, I would suggest linking to the service developer documentation, specifically the section on config options.

Reply:
Done, thank you